packages/fxa-auth-server/lib/senders/emails/partials/appBadges/index.mjml

```diff
@@ -1,38 +1,61 @@
-<%# This Source Code Form is subject to the terms of the Mozilla Public
-  # License, v. 2.0. If a copy of the MPL was not distributed with this
+<%# This Source Code Form is subject to the terms of the Mozilla Public 
+  # License, v. 2.0. If a copy of the MPL was not distributed with this 
   # file, You can obtain one at http://mozilla.org/MPL/2.0/. %>
 
 <mj-include path="./css/appBadges/index.css" type="css" css-inline="inline" />
 
+<%# TODO: Conslidate app store link names %>
+<% locals.iosUrl = locals.iosUrl || locals.appStoreLink %> 
+<% locals.androidUrl = locals.androidUrl ||  locals.playStoreLink %> 
+
 <mj-section>
-  <mj-group>
+  <mj-group css-class="app-badges">
+
+    <% if (locals.iosUrl) { %>
     <mj-column>
-      <mj-image css-class="app-badges"
+      <mj-image
+        css-class="app-badge"
         href="<%- iosUrl %>"
         src="https://accounts-static.cdn.mozilla.net/product-icons/apple-app-store.png"
-        alt="iOS badge">
+        alt="iOS badge"
+      >
       </mj-image>
     </mj-column>
+    <% } %>
+    <% if (locals.androidUrl) { %>
     <mj-column>
-      <mj-image css-class="app-badges"
+      <mj-image
+        css-class="app-badge"
         href="<%- androidUrl %>"
         src="https://accounts-static.cdn.mozilla.net/product-icons/google-play.png"
-        alt="android badge">
+        alt="android badge"
+      >
       </mj-image>
     </mj-column>
+    <% } %>
+
+    <%# Ensure bottom margin is retained. %>
+    <% if (!locals.iosUrl && !locals.androidUrl) { %>
+    <mj-column css-class="text-body">
+      <mj-text></mj-text>
+    </mj-column>
+    <% } %>
```

Comment:
Instead of an extra empty DOM element, can we conditionally set a margin-bottom class?

I think we can do something like `<mj-group css-class="app-badges <%= !locals.iosUrl && !locals.androidUrl && 'mb-5' %>">` (not tested)